### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by oliviertassinari**
This is an attempt of moving more activity on the private email channels, e.g. figma@mui.com to an open channel.

How we will scale the support on GitHub? The same way we are doing for Material UI, MUI X, and MUI Toolpad.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1.bug.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        Link to live example:
+
+        Steps:
+        1.
+        2.
+        3.
+        4.
+  - type: textarea
+    attributes:
+      label: Context 🔦
```
Comment (danilo-leal) [inline .github/ISSUE_TEMPLATE/1.bug.yml:50]: I wonder if we shouldn't put Context before expected behavior 🤔

File: .github/ISSUE_TEMPLATE/1.bug.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      description: What are you trying to accomplish? How has this issue affected you? Providing context helps us come up with a solution that is more useful in the real world.
+  - type: textarea
+    attributes:
+      label: Your environment 🌎
+      description: The version of the design file you are using. The version is usually disabled in the footer of the design frames.
```
Comment (danilo-leal) [inline .github/ISSUE_TEMPLATE/1.bug.yml:55]:
```suggestion
      description: The version of the design file you are using and, if it's Figma, whether you're using the browser or desktop web version.
```

A highlight around Figma's desktop and browser version. Also, got a bit confused by the last sentences, thus why I suggested removing it but happy to hear what you thought about it.

File: .github/ISSUE_TEMPLATE/2.feature.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        - label: I have tested the latest version
+          required: true
+  - type: textarea
+    attributes:
+      label: Summary 💡
+      description: Describe how it should work.
+  - type: textarea
+    attributes:
+      label: Examples 🌈
+      description: Provide a link to the Material Design specification, other implementations, or screenshots of the expected behavior.
```
Comment (danilo-leal) [inline .github/ISSUE_TEMPLATE/2.feature.yml:32]:
```suggestion
      description: Provide a link for appropriate benchmarks, such as Material Design or other design system's implementations, and screenshots/videos of the expected behavior.
```

**Review by danilo-leal — COMMENTED** [review]
Looking good - a couple of suggestions only.

File: .github/ISSUE_TEMPLATE/1.bug.yml (hunk 1 above)
Comment (danilo-leal) [reply .github/ISSUE_TEMPLATE/1.bug.yml:50]: Interesting. Any objections to putting that first? Context seems to be the initial thing.

**Review by danilo-leal — APPROVED** [review]
Looking good!

File: .github/ISSUE_TEMPLATE/1.bug.yml (hunk 1 above)
Comment (oliviertassinari) [reply .github/ISSUE_TEMPLATE/1.bug.yml:50]: Personally, I feel that the information is in the order of importance, the one I would use to scan a bug, first, I want to see what's the bug. But if you feel that context should be first, what do you think about proposing the change on the root issue template in https://github.com/mui/material-ui/blob/master/.github/ISSUE_TEMPLATE/1.bug.yml? What I like about the context is that I will create my own judgment on whether the expected vs. actual behavior is valid or invalid, **afterward**, I will check the context, see if this information change my worldview or reinforce my existing one.

**Pull request merged**